RevenueCatUI/CustomerCenter/ViewModels/BaseManageSubscriptionViewModel.swift

```diff
@@ -68,7 +68,7 @@ class BaseManageSubscriptionViewModel: ObservableObject {
 
     private var error: Error?
     private let loadPromotionalOfferUseCase: LoadPromotionalOfferUseCaseType
-    private let paths: [CustomerCenterConfigData.HelpPath]
+    private(set) var paths: [CustomerCenterConfigData.HelpPath]
```

Comment:
I believe this is not being set anywhere, so it can still be a `let`

Reply:
I did it so it can be accessed by the subclass. It's kind of a workaround, but it does the trick now. 
I'll most probably refactor this after the release in order to make `paths` private again, or even just compute paths differently.

Reply:
If it only need to be read but not written, then this could just be
```suggestion
    let paths: [CustomerCenterConfigData.HelpPath]
```